client/pokemon/pokemon-full.view.html

```diff
@@ -171,6 +171,7 @@ <h1 class="summary-toolbar">{{pokemon.parsedNickname}}</h1>
                   <span flex>Stat</span>
                   <span flex>Base</span>
                   <span flex>IVs</span>
+                  <span flex ng-if="pokemon.gen === 7">H/T</span>
```

Comment:
Mmm, as much as I'd like it to work on mobile, I'd be a bit hesitant to use a tooltip as I don't think they're supposed to be used for static text.